src/dotnet-scaffolding/dotnet-scaffold/AppBuilder/ScaffoldCommandAppBuilder.cs

```diff
@@ -23,7 +23,7 @@ public ScaffoldCommandApp Build()
         {
             config
                 .SetApplicationName("dotnet-scaffold")
-                .SetApplicationVersion(GetToolVersion())
+                .SetApplicationVersion(ToolHelper.GetToolVersion() ?? _backupDotNetScaffoldVersion)
```

Comment:
Since we theoretically control whether the informational version is present in the assembly or not, what's the purpose of the backup version?

Reply:
The calls to get the assembly informational version is nullable (so is ToolHelper.GetToolVersion() consequently) so its a backup to that. It's something to update every release (slightly annoying) but I can figure out a way to get it from Versions.props if that helps?